.github/workflows/gem-push.yml

```diff
@@ -0,0 +1,36 @@
+name: Ruby Gem
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Build + Publish
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    strategy:
+      matrix:
+        type:
+          - jdbc
+          - mysql
+          - postgresql
+          - redshift
+          - sqlserver
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby 2.7
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+      - name: push gem
+        uses: t3t5u/push-gem-to-gpr-action@v1
+        with:
+          language: java
+          gem-path: "embulk-output-${{ matrix.type }}/build/gems/*.gem"
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+          gradle-subproject: "embulk-output-${{ matrix.type }}"
```

Comment:
It runs as follows.
https://github.com/t3t5u/embulk-output-jdbc/actions/runs/8110152211

It will then be published as follows.
https://github.com/t3t5u/embulk-output-jdbc/pkgs/rubygems/embulk-output-sqlserver/185555226